lib/src/base-styles/_nx-colors.scss

```diff
@@ -361,6 +368,13 @@
     --nx-color-policy-threat-slider-text-disabled: var(--nx-color-policy-threat-slider-border-disabled);
     --nx-color-policy-threat-slider-text: var(--nx-swatch-indigo-97);
 
+    // NxList
+    --nx-color-list-background-hover: var(--nx-color-site-background);
+    --nx-color-list-background-selected: var(--nx-swatch-purple-30);
```

Comment:
This variable will no longer be needed when you pull latest changes from the feature branch.
We can use `--nx-color-interactive-background-selected`

Reply:
Removed here 2d430e9e98b2c4b0a0dee2215565c16aa65b7877